File: C/pares_consecutivos.c
#include <stdio.h>
#include <locale.h>

int x, soma;

int main(){
    setlocale(LC_ALL, "Portuguese");

    printf("Digite um numero inteiro: ");
    scanf("%d", &x);

    while (x != 0) {
        if (x % 2 != 0) {
            x++;
        }

        soma = 5 * x + 20;
        printf("SOMA = %d\n", soma);

        printf("Digite um numero inteiro: ");
		scanf("%d", &x);
    }

}

File: Python/pares_consecutivos.py
x = int(input("Digite um numero inteiro: "))
soma = 0

while x != 0:
    if x % 2 != 0:
        x += 1

    soma = 5 * x + 20
    print(f"SOMA = {soma}")
    x = int(input("Digite um numero inteiro: "))
